### PR TITLE
3.10: `get_event_loop` has been deprecated

### PR DESCRIPTION
**Pull request by VincentRPS**
## Summary

As shown in the docs `get_event_loop` in 3.10 raises a deprecation warning.
https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...)

### CODE REVIEW

**Comment by BobDotCom** [comment]
Is this backwards compatible with 3.8+?

**Comment by VincentRPS** [comment]
> Is this backwards compatible with 3.8+?

it should be, `get_running_loop` was added in 3.7.

**Review by Dorukyum — APPROVED** [review]

**Review by Lulalaby — DISMISSED** [review]

**Review by BobDotCom — APPROVED** [review]

File: docs/migrating_to_async.rst [hunk 1]
```diff
@@ -310,7 +310,7 @@ event loop then doing so is quite straightforward:
         yield from client.login('token')
         yield from client.connect()
 
-    loop = asyncio.get_event_loop()
+    loop = asyncio.get_running_loop()
```
Comment (BobDotCom) [inline docs/migrating_to_async.rst:313]:
```suggestion
    loop = asyncio.get_event_loop()
```
Revert conflicting change

**Review by BobDotCom — APPROVED** [review]

**Comment by Bimi05** [comment]
I recommend you use both `new_event_loop` and `set_event_loop` to the loop the latter creates because:
- `get_running_loop` returns the running loop **or raises an error if none**
in that case, you'd have to create and set a new `AbstractEventLoop`, just in case there is no running one.

**Comment by VincentRPS** [comment]
> I recommend you use both `new_event_loop` and `set_event_loop` to the loop the latter creates because:
> 
> * `get_running_loop` returns the running loop **or raises an error if none**
>   in that case, you'd have to create and set a new `AbstractEventLoop`, just in case there is no running one.

same thing happens with `get_event_loop` tho, only difference is it raises a deprecation warning instead of runtime error

**Comment by Bimi05** [comment]
> same thing happens with `get_event_loop` tho, only difference is it raises a deprecation warning instead of runtime error

Fair enough, but using it plainly would raise a `RuntimeError` if the running loop is in fact none, so it should be used within a `try`/`except` block with the concept of making and assigning a new event loop and then use it. In summary it basically is a coverage because the running loop may be none

**Comment by EmreTech** [comment]
What's the update on this? It's pretty stale by this point.

**Comment by Lulalaby** [comment]
Closing as stale

**Pull request closed without merging**